### DISCUSSION TITLE
Providers may still use Social Security Numbers

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#73] [Q&A] [answered]
There are still some providers who use a social security number rather than a Tax Identification Number for claims. 

We will not disclose the SSN due to PII, how should we populate the TIN field when the provider only provides the SSN? 

Recommendation to either leave it blank or enter all zeros
negotiated_rates": [{
      "providers": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
      "tin": "00-0000000",
or 
negotiated_rates": [{
      "providers": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
      "tin": " ",

**Comment by ghost** [comment]
Options are to

- Contact providers warning them to obtain a new TIN prior to the Transparency Rule’s requirements go into effect
- Issuers decide to mask the data themselves to avoid risk of identity theft for a provider

Our approach is to mask the TIN's of providers who are using their Social Security numbers based on directions we have received within our organization for compliance. 

A general and consistent masking approach guidance would be appreciated from CMS.

**Reply by Ginger228** [reply]
Thanks- sending a mailing out to not only prepare for Transparency but to give a nudge to update the SSN is good idea

**Comment by tdfow** [comment]
For those Providers that have a SSN versus TIN--we have a scenario that will populate the TIN information with asterisks except the last 4 digits. (For example, TIN would have *****nnnn, where nnnn equates to last 4 of digits).

**Reply by Ginger228** [reply]
Thanks. Our preference is to not display any part of the SSN.

**Comment by angelamarie0975** [comment]
I understand completely the necessity to mask the TIN if it is the SSN because of PHI data - but couldn't that potentially lead to multiple records returned since the TIN is part of the 'unique' record for the negotiated rates? Our negotiated rates are mostly contracted at the TIN level, which means if we mask that number, how can the applications using the MRF files know which contracted rate to return for the providers? 

ie. This can make a difference for a doctor that provides care in his own practice as well as be in a group of doctors and has separate contracts for each - the negotiated rates could be completely different between the 2 contracts and that would make a difference on the cost transparency being reported back to the end user. If we have provider 1234567890 with rate 1 (100.00) for the group practice (TIN 9876543210) or rate 2 (200.00) for his own practice (TIN 2345678901) - if we mask those TINs, then how would the application pulling the MRF data know which record to select if it is masked?

**Reply by ghost** [reply]
You are completely correct since both fields are required making a "and/composite key" relationship to them which their associated data is dependent on. By masking you are, as you said, obscuring the accuracy of the negotiated rate information. CMS needs to comment on this since this can be a common theme for all those generating this file knowing organizations are responsible for this information and associated compliance issues it causes.

**Reply by taylorpatriciab** [reply]
@angelamarie0975 Since this is provider data, isn't the SSN PII, not PHI?  In other words, as PII, showing it would reveal personal information but it would not fall under HIPAA regulation

**Reply by Brinkley100** [reply]
> You are completely correct since both fields are required making a "and/composite key" relationship to them which their associated data is dependent on. By masking you are, as you said, obscuring the accuracy of the negotiated rate information. CMS needs to comment on this since this can be a common theme for all those generating this file knowing organizations are responsible for this information and associated compliance issues it causes.

If we are unable to determine if a Federal Tax ID of a provider is an SSN or EIN, is anyone aware of  a service or algorithm we can use to determine if the number is an SSN or EIN?

**Reply by Ginger228** [reply]
Typically the SSN is formatted as XXX-XX-XXXX; the TIN(EIN) are XX-XXXXXXX

**Reply by Brinkley100** [reply]
Unfortunately the data that we store for  provider Tax Id  is not formatted with the dashes so makes it challenging to determine if It is EIN or SSN.

**Comment by mmorri05** [comment]
Wikipedia has some info on the structure for valid SSNs:
https://en.wikipedia.org/wiki/Social_Security_number#Structure

And for EINs:
https://en.wikipedia.org/wiki/Employer_Identification_Number#EIN_format

And the IRS details how SSNs and EINs (and other identifiers) differ and how to obtain them:
https://www.irs.gov/individuals/international-taxpayers/taxpayer-identification-numbers-tin

**Reply by Brinkley100** [reply]
Thank you for the resources.

**Comment by shaselton-usds** [comment]
Thank you for raising this issue. This is a guidance question that will be addressed directly on CMS' transparency website, but it obviously has technical implications on the implementation. Because we don't want any SSN information reported in the files for reasons already stated but want to keep that composite key to allow for uniqueness, using the provider's NPI for identification purposes would work. 

A potential implementation could look like updating the [Provider's Object](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#providers-object) from:
```json
"provider_groups": [{
  "npi": [2222222222],
  "tin": "11-1111111"
}]
```
to:
```json
"provider_groups": [{
  "npi": [2222222222],
  "tax_reporting":{
    "type": "npi",
    "id" "2222222222"
  }
}]
```

The `tax_reporting` object would have an ENUM of `npi` and `tin` and the `id` would be the value.

This would slightly change the format of TIN reporting by having a dynamic object called `tax_reporting`.

The typical use-case of a EIN/TIN that isn't a SSN would look like
```json
"provider_groups": [{
  "npi": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555],
  "tax_reporting":{
    "type": "tin",
    "id" "22-2222222"
  }
}]
```

**Reply by Ginger228** [reply]
Thanks for the response.  If the tax reporting type is SSN would this field need to be blank or would we list SSN and the ID will be left empty?

**Reply by kavkuk** [reply]
@shaselton-usds same question as @Ginger228 above.  Need to know if we need to leave SSN blank?

**Comment by shaselton-usds** [comment, marked as answer]
@Ginger228 @kavkuk @angelamarie0975 @BobSyracuse 
Please take a look at this branch: https://github.com/CMSgov/price-transparency-guide/commit/782ee80781f71773a1af19efde44d4fba0293e23#diff-9d21c67a4093a23298bf324bacbd6e6988a00a5a9001828d8cc135616e81b370

Implementation examples can be found here: https://github.com/CMSgov/price-transparency-guide/blob/ssn/examples/in-network-rates/in-network-rates-fee-for-service-sample.json#L74-L83

An explanation on how to avoid using a provider's ssn can be found here: https://github.com/CMSgov/price-transparency-guide/commit/782ee80781f71773a1af19efde44d4fba0293e23#diff-a1cdc6a3ebf570e91e3234bf152cf5978c8759e4e86f3630b83b77a115161874 

A quick overview is, always use a TIN if you have it. If the TIN is a provider's SSN, use the provider's NPI instead.

**Reply by tdfow** [reply]
@shaselton-usds Thanks for the update.  Can we officially use this guidance for the provider SSN scenarios?   If so, when will the schemas  be updated.  My assumption, is the schemas will be updated for both In-network and Allowed Amount?    

Due to the current schema, we were masking when SSN--so this now can be handled differently.

**Reply by shaselton-usds** [reply]
A technical clarification will be coming out very shortly on the TiC website on cms.gov. This technical implementation is a signal of what is to drop there.

You shouldn't treat this as official until it is posted. I'll link to the website here in this thread when it is posted.

**Reply by shaselton-usds** [reply]
@tdfow et al - you can find the official clarification here now: https://www.cms.gov/healthplan-price-transparency/resources/technical-clarification

I'll be updating the rest of the examples as well as the allowed amount file later today.

**Reply by Nio-1** [reply]
Have you updated the examples yet for both the in net and out of net schema examples?

**Reply by tdfow** [reply]
@Nio-1 @shaselton-usds   Seems the Allowed Amount Schema has not been updated yet for the TIN related changes, so do we know when to expect that schema update--- assuming it will be aligned with the In-Network related schema

**Reply by shaselton-usds** [reply]
@tdfow @Nio-1 Thanks for the watchful eye. Updates can be found in this PR: #268

**Reply by rajkumarsowmy** [reply]
@shaselton-usds 
we have a situation where we cant identify TAX_ID is SSN or Not an SSN. 
can we make the Value attribute as array as well, so we dont have to repeat this for each npi type? 
please let us know your thoughts. 


"provider_groups": [{
  "npi": [1111111111, 2222222222, 3333333333, 4444444444, 5555555555], -- _ARRAY TYPE_
  "tin":{
    "type": "npi",
    "value" "22-2222222" -- _currently it is string type, can we make it as array type , so we dont have to repeat this for each npi._
  }
}]

**Reply by rajkumarsowmy** [reply]
@shaselton-usds  Hi Scott, any thoughts around changing the value column as array?
